corehq/apps/accounting/migrations/0077_add_web_user_feature.py

```diff
@@ -0,0 +1,43 @@
+from django.db import migrations
+from django.core.management import call_command
+
+from corehq.privileges import LOADTEST_USERS
+from corehq.util.django_migrations import skip_on_fresh_install
+
+from corehq.apps.accounting.bootstrap.utils import _ensure_feature_rates
+from corehq.apps.accounting.bootstrap.config.web_user_feature_rate import BOOTSTRAP_CONFIG
+
+
+@skip_on_fresh_install
+def _grandfather_basic_privs(apps, schema_editor):
+    call_command('cchq_prbac_bootstrap')
+    call_command(
+        'cchq_prbac_grandfather_privs',
+        LOADTEST_USERS,
+        skip_edition='Paused,Community',
+        noinput=True,
+    )
+
+
+def _add_web_user_feature(apps, schema_editor):
+    Feature = apps.get_model('accounting', 'Feature')
+    web_user_feature = Feature.objects.create(name='Web User', feature_type='Web User')
+    web_user_feature.save()
```

Comment:
This save is redundant. `Feature.objects.create(name='Web User', feature_type='Web User')` will create and save the entry in database.

Reply:
Addressed here: 20f4ae094dfbeff463c1d837a30b095b8cccc076